### ISSUE TITLE
Blocks are too close to left-edge in toolbox flyout

### ISSUE THREAD

**Issue by gezakerecsenyi** [codeddraig/ffau#88]
In the toolbox flyout, the blocks are very close to the left edge (to the section selector), and it looks awkward and distracting as it overlaps with the shadow.

This can be fixed by adding:

```css
.blocklyFlyout g.blocklyBlockCanvas {
        transform: translate(10px, 10px)
}
```

**Comment by palkerecsenyi** [comment]
This breaks scrolling and looks quite odd:

![image](https://user-images.githubusercontent.com/22198767/76151045-51b7e580-60a8-11ea-8161-942fa3af9dfe.png)

**Comment by palkerecsenyi** [comment]
Issue has been fixed Blockly `3.20200123.0` but this also breaks code generation completely. I've submitted an issue to Blockly here: https://github.com/google/blockly/issues/3794

**Comment by palkerecsenyi** [comment]
Solved by Blockly's new patch
